tests/test_validate.py

```diff
@@ -31,29 +25,22 @@ def _predict(self, item):
     valid_test_item = {"x": 10}
```

Comment:
NB: for this testing file, if instead you build a valid_test_item with `ItemModel(x=10)`, you won't have to add model_dump further in the different asserts, will you ?